### PR TITLE
Profane -> argparse

### PR DESCRIPTION
**Pull request by thongnt99**
Example: 
python -m diffir.run robust04/maxp_electra-robust04_title.run robust04/cedr-knrm.run -w --dataset trec-robust04 --measure metric  --topk 200 --metric=tauap --weights_1 robust04/cedr-knrm.diff --weights_2 robust04/maxp_electra-robust04_title.run.diffir

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install numpy
+          pip install scipy
+          pip install pandas
```
Comment (andrewyates) [inline .github/workflows/python-package.yml]: I think we need to add scipy/numpy/pandas to `requirements.txt`?

File: diffir/weight/unsupervised.py [hunk 2]
```diff
@@ -3,20 +3,16 @@
...
-    config_spec = [
-        ConfigOption(key="skip_stopwords", default_value=True, description="TODO"),
-        ConfigOption(
-            key="queryfield", default_value="", value_type="strlist", description="The query field that is used for highlighting"
-        ),
-    ]
+
+    def __init__(self, queryfield="", skip_stopwords=True):
+        self.queryfield = queryfield
+        self.skip_stopwords = True
```
Comment (andrewyates) [inline diffir/weight/unsupervised.py]: I think this should be `self.skip_stopwords = skip_stopwords` ?

**Review by andrewyates — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (thongnt99) [reply .github/workflows/python-package.yml]: I added

File: diffir/weight/unsupervised.py (hunk 2 above)
Comment (thongnt99) [reply diffir/weight/unsupervised.py]: this is fixed

**Review by andrewyates — APPROVED** [review]

**Review by andrewyates — APPROVED** [review]

**Pull request merged**